functions/cloudflare_b2_proxy.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Cloudflare Workers script to proxy requests to Backblaze B2 with 
+ * CORS headers and key from environment variables, to sign URLs
+ *
+ * Learn more at https://developers.cloudflare.com/workers/
+ */
+
+export default {
+    async fetch(request, env, ctx) {
+      const corsHeaders = {
+        'Access-Control-Allow-Origin': 'https://app.kendra.io',
+        'Access-Control-Allow-Methods': 'GET, OPTIONS',
+        'Access-Control-Allow-Headers': 'Content-Type',
+      };
+  
+      // Handle OPTIONS request
+      if (request.method === 'OPTIONS') {
+        return new Response(null, { headers: corsHeaders });
+      }
+  
+      const url = new URL(request.url);
+      
+      // Return hello world if no path specified
```

Comment:
This doesn't seems very helpful to me.
Can we not return random text which can result in confusion? 
I would suggest to either return an error or a message like "please specify path" (if we want people to have hints)